services/src/main/java/org/jboss/windup/web/services/model/AnalysisContext.java

```diff
@@ -273,6 +276,21 @@ public void removeApplication(RegisteredApplication application)
         this.getApplications().remove(application);
     }
 
+    public AnalysisContext clone()
+    {
+        AnalysisContext clone = new AnalysisContext(this.migrationProject);
```

Comment:
what do you mean by evicting from session? You mean detach it from entity manager, set id to NULL and persist?